enginetest/enginetests.go

```diff
@@ -1699,6 +1698,21 @@ func TestRenameColumn(t *testing.T, harness Harness) {
 		[]sql.Row{
 			{1, "first row"},
 		}, nil, nil)
+
+	// Test column rename behavior when a table check constraint is involved
+	RunQuery(t, e, harness, "ALTER TABLE mytable ADD CONSTRAINT test_check CHECK (i2 < 12345)")
```

Comment:
Should enclose this new section in a named `t.Run()`

Reply:
✅